pkg/commands/profile/token.go

```diff
@@ -60,3 +73,21 @@ func (c *TokenCommand) Exec(_ io.Reader, out io.Writer) (err error) {
 		Remediation: fsterr.ProfileRemediation,
 	}
 }
+
+func checkTokenValidity(profileName string, p *config.Profile, ttl int64) (err error) {
```

Comment:
Per my message above, this can/should accept a `time.Duration` for the `ttl`.

```suggestion
func checkTokenValidity(profileName string, p *config.Profile, ttl time.Duration) (err error) {
```